modules/@apostrophecms/ui/ui/apos/components/AposCellLabels.vue

```diff
@@ -39,6 +39,9 @@
         v-if="evaluateConditions(conditions)"
         class="apos-table__cell-field__label"
         v-bind="props"
+        :header="header"
+        :draft="draft"
+        :published="published"
```

Comment:
Should be available to the custom component so it can compute its visibility via client code and not static configuration. Also `header` is backend configuration passed to the component.